### PR TITLE
Integration canary deploy

### PR DESCRIPTION
**Pull request by tcoupland**
Adding various files and scripts to enable integration test execution

### CODE REVIEW

File: deployment/check_tests.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# replace dots with hyphens in APP_NAME
+APP_NAME=$1
+SEBASTOPOL_IP=$2
+
+TEST_FILE=test-output
```
Comment (acron0) [inline deployment/check_tests.sh]: TEST_FILE not used

File: src/witan/workspace_peer/config.clj [hunk 2]
```diff
@@ -1,11 +1,11 @@
 (ns witan.workspace-peer.config
   (:require [witan.workspace-peer.utils.workflow :refer :all]
             ;;
-            [witan.models.dem.ccm.fert.fertility-mvp]))
+            [witan.models.dem.ccm.fert.fertility]))
```
Comment (acron0) [inline src/witan/workspace_peer/config.clj:4]: I assume the model library version has been updated to require this?

Comment (tcoupland) [inline src/witan/workspace_peer/config.clj:4]: it's a snapshot version, so it looks like it the namespace name just got trimmed and the snapshot got updated.

**Comment by acron0** [comment]
:shipit:

**Pull request merged**